src/http/get-create_part_v2/index.js

```diff
@@ -35,6 +35,8 @@ exports.handler = async (req) => {
   const urls = {}
   const roots = []
   const dagBlocks = []
+  // iterate over each file collecting the blocks and generating
+  // root block referencing each part
```

Comment:
it doesn’t “iterate over each file” in that it’s not actually looking at file data, it’s just looking at the list of block addresses for each part. the `_parts` are just string CIDs and not the actual data.

later you can see that we actually have to retrieve the block data by address for every part out of the block store bucket.